packages/editor-ui/src/plugins/codemirror/completions/datatype.completions.ts

```diff
@@ -151,31 +162,96 @@ export const toOptions = (fnToDoc: FnToDoc, typeName: ExtensionTypeName) => {
 				tooltipContainer.style.paddingTop = 'var(--spacing-4xs)';
 				tooltipContainer.style.paddingBottom = 'var(--spacing-4xs)';
 
-				const header = document.createElement('div');
-				header.style.marginBottom = 'var(--spacing-2xs)';
+				const header =
+					optionType === 'function'
+						? createFunctionHeader(typeName, fn)
+						: createPropHeader(typeName, fn);
+				tooltipContainer.appendChild(header);
 
-				const typeNameSpan = document.createElement('span');
-				typeNameSpan.innerHTML = typeName.slice(0, 1).toUpperCase() + typeName.slice(1) + '.';
+				const descriptionBody = document.createElement('div');
+				const descriptionText = document.createElement('p');
+				descriptionText.style.marginTop = '0';
+				descriptionText.style.lineHeight = '1.1em';
+				descriptionText.style.marginBottom = 'var(--spacing-2xs)';
+				descriptionText.innerHTML = sanitizeHtml(
+					fn.doc.description.replace(/`(.*?)`/g, '<code>$1</code>'),
+				);
+				descriptionBody.appendChild(descriptionText);
+				if (fn.doc.docURL) {
+					const descriptionLink = document.createElement('a');
+					descriptionLink.setAttribute('target', '_blank');
+					descriptionLink.setAttribute('href', fn.doc.docURL);
+					descriptionLink.innerText = i18n.autocompleteUIValues['docLinkLabel'] || 'Learn more';
+					descriptionLink.addEventListener('mousedown', (event: MouseEvent) => {
+						event.preventDefault();
```

Comment:
what's this for?

Reply:
This will prevent documentation link clicks to close autocomplete dropdown before link opens.

Reply:
Can you add a comment to clarify

Reply:
Added.